src/components/editor/inputs/InputLiteralValue.jsx

```diff
@@ -69,7 +70,9 @@ const InputLiteralValue = ({
     handleKeyDownDiacritics(event)
   }
 
-  const showLang = !propertyTemplate.languageSuppressed || value.lang
+  const showLang =
+    (!propertyTemplate.languageSuppressed || value.lang) &&
+    !propertyTemplate.validationDataType
 
```

Comment:
don't show the language field if we have validating a type (e.g. integer or datetime) as noted in the ticket